mlflow/genai/__init__.py

```diff
@@ -1,4 +1,15 @@
 from mlflow.genai.evaluation import evaluate, to_predict_fn
 from mlflow.genai.scorers import Scorer, scorer
 
-__all__ = ["evaluate", "to_predict_fn", "Scorer", "scorer"]
+try:
+    from databricks.agents.review_app import (
+        ReviewApp,
+        get_review_app,
+    )
+except ImportError:
+    raise ImportError(
+        "The `databricks-agents` package is required to use `mlflow.genai.ReviewApp` and `mlflow.genai.get_review_app`. "
+        "Please install it with `pip install databricks-agents`."
+    )
```

Comment:
@smoorjani @B-Step62 This will break imports of `mlflow.genai` if `databricks-agents` is not installed, which we don't want to do.

Instead of this approach, I'd prefer that we make every databricks-agents API importable, and raise errors when users try to call them without having databricks-agents SDK installed. Yes, this will require wrapper functions, but it's a significantly better devex because users who forget to install databricks-agents SDK don't get "function not found" errors or wonder why their autocomplete is suddenly returning zero results, etc.

@smoorjani Once we make this change, can we import *everything* except the contents of `databricks.agents.review_app.label_schemas` into `mlflow.genai` for improved ease of access?